### PR TITLE
Extend alternate item description to 255 characters.

### PR DESCRIPTION
**Pull request by SteveIreland**
This pull request introduces the changes needed to allow for an alternate item description of up to 255 characters.

The alternate description is used to provide a more complete description of the service/labor item to be invoiced.

### CODE REVIEW

File: application/views/sales/invoice.php [hunk 1]
```diff
@@ -108,21 +108,26 @@ function(response)
...
-				<td style='text-align:center;'><textarea rows="4" cols="6"><?php echo $item['discount'] .'%'; ?></textarea></td>
-				<td style='border-right: solid 1px; text-align:right;'><textarea rows="4" cols="6"><?php echo to_currency($item['discounted_total']); ?></textarea></td>
-			</tr>
-			<tr class="item-row">
-				<td></td>
-				<td class="item-name"><textarea cols="6"><?php echo $item['description']; ?></textarea></td>
-				<td style='text-align:center;' colspan="4"><textarea cols="6"><?php echo $item['serialnumber']; ?></textarea></td>
-			</tr>
-		<?php
+			?>
```
Comment (jekkos) [inline application/views/sales/invoice.php]: Some indentation issues here?

Comment (jekkos) [inline application/views/sales/invoice.php]: Kind of hard to see what changed now

Comment (daN4cat) [inline application/views/sales/invoice.php]: it was neat before now it's kind of messy.... :-(

Comment (SteveIreland) [inline application/views/sales/invoice.php]: yes.  I'll transform the indents to tabs and reupload it.  That might help a bit.

**Comment by SteveIreland** [comment]
Hopefully this is a bit better.  I corrected some misaligned <?php  - ?> combinations and tried my best to clean up any indentation issues.

**Comment by daN4cat** [comment]
Does the invoice in pdf work? I have vague memory that changing format in the invoice was causing issue with the pdf generator but it might be an old story... I'll test before merging anyway.

**Comment by jekkos** [comment]
Invoice PDF generation is pretty brittle indeed.. it breaks easily when changing CSS etc

**Comment by daN4cat** [comment]
I merged but the invoice is throwing an error in the pdf when sent over email.

I don't have time to debug, @SteveIreland if you could check.

**Pull request closed without merging**